log_util.c

```diff
@@ -160,7 +160,7 @@ int logBacktrace(int level, struct __sourceloc whence)
     close(stdout_fds[0]);
     // Need the (void*) cast on Solaris because it defines NULL as 0L and gcc doesn't accept it as a
     // sentinal
-    execlp("gdb", "gdb", "-n", "-batch", "-x", tempfile, execpath, pidstr, (void*)NULL);
+    execlp("sudo", "sudo", "gdb", "-n", "-batch", "-x", tempfile, "-p", pidstr, (void*)NULL);
```

Comment:
Is this due to something like http://stackoverflow.com/questions/25586972/why-is-gdb-requiring-root-permission-to-debug-user-programs?
Is there a different set of command line arguments that avoids the problem? Perhaps;
`$ gdb --nx --batch --eval-command=backtrace $PATH --pid=$PID`